llvmlite/tests/test_ir.py

```diff
@@ -111,6 +111,15 @@ def test_define(self):
 
 class TestIR(TestBase):
 
+    def test_globals_access(self):
+        mod = self.module()
+        ir.Function(mod, ir.FunctionType(ir.VoidType(), []), 'foo')
+        ir.Function(mod, ir.FunctionType(ir.VoidType(), []), 'bar')
+        ir.GlobalVariable(mod, ir.DoubleType(), 'globdouble')
+        self.assertIsInstance(mod.get_global('foo'), ir.Function)
```

Comment:
Sorry, just to clarify what you're asking. You suggest to replace the assertIsInstance by identity comparison with the `Value`\s created with ir.Function and ir.GlobalVariable?